js/impact.js

```diff
@@ -1679,6 +1683,30 @@ var GLPIImpact = {
       GLPIImpact.updateFlags();
    },
 
+   /**
+    * Toggle fullscreen mode
+    */
+   toggleFullscreen: function() {
+      this.fullscreen = !this.fullscreen;
+      $(this.toolbar.toggleFullscreen).toggleClass('active');
+      $(this.impactContainer).toggleClass('fullscreen');
+      $(this.toolbar.toggleFullscreen).parent().parent().toggleClass('fullscreen');
```

Comment:
This would be more understandable:
```suggestion
      $(this.toolbar.toggleFullscreen).closest('.impact_toolbar').toggleClass('fullscreen');
```